--- config.py   2015-05-16 05:33:18.506156647 +0000
+++ newconfig.py        2015-05-16 05:31:42.571734424 +0000
@@ -17,6 +17,7 @@
 from ConfigParser import ConfigParser
 from copy import deepcopy
 import os.path
+import os

 from genshi.builder import tag
 from trac.admin import AdminCommandError, IAdminCommandProvider
@@ -60,7 +61,7 @@
     """
     def __init__(self, filename, params={}):
         self.filename = filename
-        self.parser = ConfigParser()
+        self.parser = ConfigParser(os.environ)
         self._old_sections = {}
         self.parents = []
         self._lastmtime = 0
